src/test/java/seedu/address/logic/parser/SortStudentCommandParserTest.java

```diff
@@ -27,30 +28,37 @@ public void parse_emptyArg_throwsParseException() {
     @Test
     public void parse_validArgs_returnsSortStudentCommand() {
         // valid name
-        SortStudentCommand command = new SortStudentCommand("name");
+        SortStudentCommand command = new SortStudentCommand(SortStudentParameter.NAME);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "name", command);
 
         // valid name
-        command = new SortStudentCommand("id");
+        command = new SortStudentCommand(SortStudentParameter.STUDENTID);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "id", command);
 
         // valid name
-        command = new SortStudentCommand("email");
+        command = new SortStudentCommand(SortStudentParameter.EMAIL);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "email", command);
 
         // case-insensitive
-        command = new SortStudentCommand("name");
+        command = new SortStudentCommand(SortStudentParameter.NAME);
         assertParseSuccess(parser, " " + PREFIX_SORT_BY + "NaMe", command);
 
         // extended preamble
-        command = new SortStudentCommand("id");
+        command = new SortStudentCommand(SortStudentParameter.STUDENTID);
         assertParseSuccess(parser, "      " + PREFIX_SORT_BY + "id      ", command);
     }
```

Comment:
Is this comment updated?

Reply:
Good catch!

Reply:
This assertion is used to test the extra white spaces in the preamble of the command. I will update it to include that.